hack/dump-state.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+
+function RunCmd {
+    cmd="$@"
+    echo "Command: $cmd"
+    $cmd
+    if [ "x$?" != "x0" ]; then
+        echo "Command failed: $cmd Status: $?"
+    fi
+}
+
+cat <<EOF
+=================================
+     Start of HCO state dump         
+=================================
+
+========
+HCO Pods
+========
+
+EOF
+
+RunCmd "$CMD get pods -n kubevirt-hyperconverged -o json"
+
+cat <<EOF
```

Comment:
I would also add a section for that HCO's status. You can use `oc get hco -n kubevirt-hyperconveged hyperconverged-cluster -o=jsonpath='{range .status.conditions[*]}{.type}{"\t"}{.status}{"\t"}{.message}{"\n"}{end}'` to get the conditions.

Reply:
ok will add that.